### ISSUE TITLE
Add summary table of sites and other site info to docs

### ISSUE THREAD

**Issue by eteq** [astropy/astroplan#40]
The sites functionality in #26 only has API docs - there need to be some narrative docs describing how it's used.  @cdeil suggested in #26 that there should be an auto-generated table of the observatories that are built-in (I think just the names/aliases/description/source are sufficient for the docs).

**Comment by cdeil** [comment]
@jberlanga – Maybe you could address this issue in #42 ?

**Comment by eteq** [comment]
I agree with @cdeil about the *narrative* part being in #42, but it will be a bit more complicated to auto-generate a table (that requires adding code to the module itself). So I suggest treating that as a separate PR after #42

**Comment by jberlanga** [comment]
@cdeil @eteq  I'll add a section **about** the sites (with @bmorris3's input) for now. in PR #42

**Comment by bmorris3** [comment]
Let me know if/when you need help with the sites docs, @jberlanga.

**Comment by jberlanga** [comment]
@astropy/astroplan-maintainers -- Apologies for leaving this issue by the wayside! Moving this to 0.2 (unless someone gets to it before 0.1 drops).

**Comment by adrn** [comment]
I have an idea for how to do this -- if we convert the `observatories.json` file into a CSV, we can use:
http://docutils.sourceforge.net/docs/ref/rst/directives.html#csv-table

But I want all of the longitudes/latitudes to be numbers so I can easily generate Google maps links for each row in the table...

**Comment by bmorris3** [comment]
Now that #105 is merged, might you be able to attack this @adrn?

**Comment by bmorris3** [comment]
Since astropy/astropy#4042, the `sites` module has been ~ported to astropy, making these docs unnecessary in astroplan (though they'd still be great in astropy). When `sites` is removed in favor of the astropy implementation, this issue should be closed.

**Comment by bmorris3** [comment]
Since I'm removing the `sites` module in #168, I'm closing this issue.
